Build failure: libsForQt5.qt5.qtwebengine

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build libsForQt5.qt5.qtwebengine

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
|              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |              AV_PROFILE_H264_HIGH_444_PREDICTIVE
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:282:10: error: 'FF_PROFILE_UNKNOWN' was not declared in this scope; did you mean 'AV_PROFILE_UNKNOWN'?
  282 |   return FF_PROFILE_UNKNOWN;
      |          ^~~~~~~~~~~~~~~~~~
      |          AV_PROFILE_UNKNOWN
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc: In function 'bool media::AVCodecContextToAudioDecoderConfig(const AVCodecContext*, EncryptionScheme, AudioDecoderConfig*)':
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:418:55: error: 'FF_PROFILE_UNKNOWN' was not declared in this scope; did you mean 'AV_PROFILE_UNKNOWN'?
  418 |   if (codec == kCodecAAC && codec_context->profile == FF_PROFILE_UNKNOWN ||
      |                                                       ^~~~~~~~~~~~~~~~~~
      |                                                       AV_PROFILE_UNKNOWN
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc: In function 'bool media::AVStreamToVideoDecoderConfig(const AVStream*, VideoDecoderConfig*)':
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:540:14: error: 'FF_PROFILE_VP9_0' was not declared in this scope; did you mean 'AV_PROFILE_VP9_0'?
  540 |         case FF_PROFILE_VP9_0:
      |              ^~~~~~~~~~~~~~~~
      |              AV_PROFILE_VP9_0
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:543:14: error: 'FF_PROFILE_VP9_1' was not declared in this scope; did you mean 'AV_PROFILE_VP9_1'?
  543 |         case FF_PROFILE_VP9_1:
      |              ^~~~~~~~~~~~~~~~
      |              AV_PROFILE_VP9_1
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:546:14: error: 'FF_PROFILE_VP9_2' was not declared in this scope; did you mean 'AV_PROFILE_VP9_2'?
  546 |         case FF_PROFILE_VP9_2:
      |              ^~~~~~~~~~~~~~~~
      |              AV_PROFILE_VP9_2
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:549:14: error: 'FF_PROFILE_VP9_3' was not declared in this scope; did you mean 'AV_PROFILE_VP9_3'?
  549 |         case FF_PROFILE_VP9_3:
      |              ^~~~~~~~~~~~~~~~
      |              AV_PROFILE_VP9_3
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:648:15: error: 'const AVStream' {aka 'const struct AVStream'} has no member named 'nb_side_data'
  648 |   if (stream->nb_side_data) {
      |               ^~~~~~~~~~~~
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:649:33: error: 'const AVStream' {aka 'const struct AVStream'} has no member named 'nb_side_data'
  649 |     for (int i = 0; i < stream->nb_side_data; ++i) {
      |                                 ^~~~~~~~~~~~
../../3rdparty/chromium/media/ffmpeg/ffmpeg_common.cc:650:44: error: 'const AVStream' {aka 'const struct AVStream'} has no member named 'side_data'
  650 |       AVPacketSideData side_data = stream->side_data[i];
      |                                            ^~~~~~~~~
^M[10306/24855] ACTION //services/network/public/mojom:mojom_network_isolation_key__generator(/build/source/src/toolchain:target)^M[10307/24855] ACTION //services/network/public/mojom:mojom_i>
ninja: build stopped: subcommand failed.
make[3]: *** [Makefile.gn_run:372: run_ninja] Error 1
make[3]: Leaving directory '/build/source/src/core'
make[2]: *** [Makefile:82: sub-gn_run-pro-make_first] Error 2
make[2]: Leaving directory '/build/source/src/core'
make[1]: *** [Makefile:80: sub-core-make_first] Error 2
make[1]: Leaving directory '/build/source/src'
make: *** [Makefile:49: sub-src-make_first] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.57, NixOS, 25.11 (Xantusia), 25.11.20251111.9da7f1c`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/cjjpqrp9xy44j5fjh3mygyxy3i7mslj2-source`

### Notify maintainers

@matthewbauer 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/libraries/qt-5/modules/qtwebengine.nix
+++ b/pkgs/development/libraries/qt-5/modules/qtwebengine.nix
@@ -57,7 +57,7 @@
   bootstrap_cmds,
   xcbuild,
   writeScriptBin,
-  ffmpeg ? null,
+  ffmpeg_7 ? null,
   lib,
   stdenv,
   version ? null,
@@ -354,7 +354,7 @@ qtModule (
       icu
 
       libevent
-      ffmpeg
+      ffmpeg_7
     ]
     ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [
       dbus

